### PR TITLE
Feat: View mode dropdown

### PR DESCRIPTION
**Pull request by marek-mihok**
https://user-images.githubusercontent.com/23740173/173614266-c173a5d6-dc1b-466a-b28b-c03c06006e1d.mov

### CODE REVIEW

File: web-ide/web_ide.py [hunk 1]
```diff
@@ -213,13 +213,19 @@ async def serve(q: Q):
 
     if q.args.export:
         await export(q)
-    elif q.args.code:
+    elif q.args.dropdown == 'code':
+        q.user.view = "code"
+        q.page['header'].items[3].dropdown.value = 'code'
```
Comment (mturoci) [inline web-ide/web_ide.py:218]: Can we do something more general like
```suggestion
    elif q.args.dropdown:
        q.user.view = q.args.dropdown
        q.page['header'].items[3].dropdown.value = q.args.dropdown
```

Would require changing elifs to ifs though.

File: web-ide/web_ide.py [hunk 2]
```diff
@@ -83,13 +83,13 @@ async def setup_page(q: Q):
         subtitle='Develop Wave apps completely in browser',
         image='https://wave.h2o.ai/img/h2o-logo.svg',
         items=[
-            ui.button(name='console', label='Console', icon='CommandPrompt'),
+            ui.button(name='console', label='Console', value='split', icon='CommandPrompt'),
```
Comment (mturoci) [inline web-ide/web_ide.py:86]: What is the `value` here for?

Comment (marek-mihok) [reply web-ide/web_ide.py:86]: I just forgot to remove it

**Review by mturoci — APPROVED** [review]
Thanks @marek-mihok!

**Pull request merged**